src/HatchIntakeSubsystem.cpp

```diff
@@ -78,13 +58,28 @@ void HatchIntakeSubsystem::SetHatchActuator(bool goUp) {
     }
     else {
         if (m_hatchActuatorDeadband.Get() <= abs(m_actuatorBottomLimit.Get() - m_hatchActuator.GetSelectedSensorPosition())) {
-        m_hatchActuator.Set(ControlMode::PercentOutput, 0.2);
+        m_hatchActuator.Set(ControlMode::PercentOutput, -0.2);
         } else {
             m_hatchActuator.Set(ControlMode::PercentOutput, 0.0);
         }
     }
+
+    if(setIntakeDirection) {
+        if(isHatchIn) {
```

Comment:
For this I think that you could just call the HatchIn() method in this class instead of adding an additional parameter for isHatch in.